tools/TM-align/compile.sh

```diff
@@ -0,0 +1,11 @@
+# Download test data
+mkdir -p testdata
+cd testdata
+wget -c "https://zhanggroup.org/TM-align/example/strA.pdb" "https://zhanggroup.org/TM-align/example/strB.pdb"
+cd -
+
+# Download code and compile it
+wget -c "https://zhanggroup.org/TM-align/TMalign.cpp"
+em++ -static -O3 -ffast-math -lm $EM_FLAGS \
```

Comment:
I get this error during compilation:

```c
TMalign.cpp:5141:14: error: expected unqualified-id
             else if (ylen<3)
             ^
TMalign.cpp:5146:21: error: C++ requires a type specifier for all declarations
                    NewArray(&ya, ylen, 3);
                    ^
TMalign.cpp:5146:31: error: use of undeclared identifier 'ya'
                    NewArray(&ya, ylen, 3);
                              ^
TMalign.cpp:5146:35: error: use of undeclared identifier 'ylen'
                    NewArray(&ya, ylen, 3);
                                  ^
```

Any thoughts?

Reply:
It works when I used ```emcc v3.1.0```  like this:
![1701653168603](https://github.com/biowasm/biowasm/assets/12126846/0000daf9-f5e6-4cb5-9068-f950e49a8090)
what 's the  version of ```emcc```of yours?

Reply:
Ah that's probably why, biowasm uses Emscripten v2—see [this page for details](https://github.com/biowasm/biowasm/blob/main/CONTRIBUTING.md#development-setup). Are you able to use that setup? Thanks!

Reply:
If above error mean that ```TMalign.cpp``` need new version of emcc, but I don't known how to  modify the cpp code to compile with ```emcc v2```,  the author release the code at [this](https://zhanggroup.org/TM-align/).

Reply:
how about modify the ```compile.sh``` to like this:   if the ```emcc version``` is suitable(example ```v3```) to compile for this ```TMalign``` then compile , but if the ```emcc version``` is not suitable then ```just not compile``` and use ```the wasm file``` which I compiled at my local computer ?  because the ```emcc v2``` will be old for more software in the future.

Reply:
ok, have any time plan to update to emcc v3 yet?

Reply:
I don't have a timeframe yet but definitely in 2024 :)

Reply:
ok

Reply:
should I first close this PR now？